### DISCUSSION TITLE
Uploading proof of enrolment does not work

### DISCUSSION THREAD

**Discussion by ghost** [community/community#109821] [GitHub Education] [answered]
### Hi everyone,

I made an application for GitHub education and was rejected, giving me this steps to solve it:

- You must secure your GitHub account by enabling 2FA and completing your billing information.
- Please secure your GitHub account with two-factor authentication. You may need to log out and log back in to GitHub before re-applying.
- You are unlikely to be verified until you have completed your GitHub billing information with your full name exactly as it appears in your academic affiliation document. If you have only a single legal name, enter it in both the first and last name fields. You do not have to add a payment method. You may need to log out and log back in to GitHub before re-applying.
- Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.

After doing all of this, it no longer shows me these errors but when I upload the proof and click on "Process my application", it just reloads the same page. And when I go to check my application status, it just shows this page of the step of uploading proof.

Any help would be very much appreciated, thanks.

![image](https://github.com/community/community/assets/160319631/d71f46e9-0228-423a-b9d8-ef171f381b28)

**Comment by itsUdit24** [comment]
I'm myself facing the same issue

**Reply by ghost** [reply]
As @martinj80 pointed out: check that the file size is <1MB

**Reply by itsUdit24** [reply]
my file size is lesser than 1MB but still it's not happening

**Reply by ghost** [reply]
Then I'm sorry. Guess you'll have to open a post or ticket to support. Good luck!

**Reply by nsitu** [reply]
FYI @itsUdit24 I had better success using the webcam instead of uploading a pre-existing JPEG because GitHub deems it more "trustworthy".

**Comment by martinj80** [comment, marked as answer]
I was also having the same problem. Is your image file <1MB?

**Reply by ghost** [reply]
OMG! I'm an idiot. Yes, it is. I'm gonna try with reduced size. Thanks for pointing it out!! (although, why TH don't they allow at leas 4 or 5 MB?? Or point it out at least?)

ETA: It worked!

**Comment by nsitu** [comment]
I had this problem as well. Thanks @martinj80  for pointing out the 1MB limit which was not well advertisted or validated (no notice was given upon incorrect submission as to why it was incorrect). After all that I found that GitHub still detected "manipulation" in the JPEG I uploaded initially. The only "manipulation" was to crop the image down to 1MB 🤣.  The solution for me was to use the webcam to take the photo instead of uploading an existing one; GitHub deems this to be more "trustworthy".

**Comment by sabghat90** [comment]
Facing same problem, even doc size is >1MB
